doc/articles/wpf-migration.md

```diff
@@ -0,0 +1,60 @@
+
+![A WPF app migrated to the web with Uno platform](Assets\migrated-app.png)
+
+## Introduction
+
+While improvements will continue to be made by Microsoft at the framework level to ensure the compatibility and reliability of your WPF application, it may be desirable to expand its reach. In this hybrid work era, previous assumptions and development decisions which served your customers well become less valid. For instance, applications targeting the Windows platform alone miss out on mobility benefits of the Web. 
+
+Advancements such as WebAssembly (WASM), a low-level bytecode for the web, have already become a viable, performant way to deliver your modern .NET applications.
+
+## Migrating the codebase
+
+The Uno Platform not only allows you to maximize reuse of C#, XAML, and .NET investments, but also enables your team to naturally leverage their existing skillset. Alongside your existing investments, you may decide to further enhance your app, refreshing aspects to make use of new capabilities in the platform. Regardless, your goal is to consider what role your application will play several years or more into the future. Depending on your scenario, it may make sense to enhance the user interface layout for different form factors. In this article, the primary focus will be on rewriting as little as possible.
+
+> [!TIP]
+> Read the UWP/WinUI & Uno documentation first to better understand the capabilities of the platform. 
+
+## Limitations
+
+The vast majority of your codebase will simply "move across". Existing XAML structure, business logic, and view models can simply be reused. You'll need to recompile old code, and won't have access to framework-only APIs. You may encounter APIs that aren't implemented or controls where a subset of the functionality is presently unsupported. Additionally, the following contain substantial enough differences from WPF to require extra research upfront:
+* Navigation
+* Advanced binding functionality with x:Bind
+* Specific framework capabilities
+* Modern APIs are async only
+* Uno has a streamlined set of XAML capabilities for performance reasons
+
+## Understanding beforehand
+
+Along the journey to bring your WPF app to the Web, you should become familiar with the Uno Platform documentation. Your team should thoroughly understand what is supported (and where) before you build a feature around it. Learn about:
+* Get started
+* How-tos and tutorials
+* Development deep-dives
+* Reference
+
+## Architecture Considerations
+
+### MVVM Libraries
+Several of the popular MVVM libraries like MVVMLite, Prism, and the Windows Community Toolkit implementation have already been ported to Uno. If your existing app leverages something in-house, there's a chance it will just work. All in all, Uno has made significant strides to improve porting MVVM applications.
```

Comment:
```suggestion
Several of the popular MVVM libraries like MVVMLite, Prism, and the Windows Community Toolkit implementation have already been ported to Uno. If your existing app leverages something in-house, there's a chance it will just work. 
```

Removed a sentence that would be fine in a blog post (or a talk), but doesn't really fit in the context of documentation. ('Significant strides' since when? How is this actionable by the reader?)